honeypot.admin.inc

```diff
@@ -42,7 +44,7 @@ function honeypot_admin_form($form, &$form_state) {
   $form['configuration']['honeypot_time_limit'] = array(
     '#type' => 'textfield',
     '#title' => t('Honeypot time limit'),
-    '#description' => t('Minimum time required before form should be considered entered by a human instead of a bot. Set to 0 to disable.'),
+    '#description' => t('Minimum time required before form should be considered entered by a human instead of a bot. A reasonable time-limit is 5s. Set to 0 to disable.'),
```

Comment:
I think changing "5s" to "5 seconds" is clearer and people are less likely to try type in `5s` into the field.

Reply:
good recommendation :)